tools/building.py

```diff
@@ -539,6 +539,7 @@ def transpile(filename):
   config_json = json.dumps(config, indent=2)
   outfile = shared.get_temp_files().get('babel.js').name
   config_file = shared.get_temp_files().get('babel_config.json').name
+  logger.debug(config_json)
```

Comment:
temp debugging?

Reply:
I actually want to be able to see the config file contents when I do `EMCC_DEBUG=1` .. its only about 10 lines.